distribution/test/itests/pom.xml

```diff
@@ -40,11 +42,6 @@
                 <version>${project.version}</version>
             </dependency>
             <dependency>
-                <groupId>org.hamcrest</groupId>
```

Comment:
This is required for our itests. We redeploy hamcrest as a bundle so we can use the full set of matchers instead of what pax-exam provides.
https://github.com/codice/ddf/blob/master/distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/AbstractIntegrationTest.java#L412